tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -49,6 +47,11 @@ public static class Tags
         /// </summary>
         public const string HttpMethod = "http.method";
 
+        /// <summary>
+        /// Framework specific key, that is the resource part of the url with parameters replaced by place holders
+        /// </summary>
+        internal const string EndPoint = "http.endpoint";
```

Comment:
Do we need both EndPoint and HttpEndpoint?

Reply:
Yes, correct removed in 2403b6fd